core/deployment/src/main/java/io/quarkus/deployment/builditem/AppModelProviderBuildItem.java

```diff
@@ -27,6 +33,20 @@ public AppModelProviderBuildItem(ApplicationModel appModel, Supplier<DependencyI
         this.depInfoProvider = depInfoProvider;
     }
 
+    /**
+     * Validates the platform imports in the application model against the provided bootstrap configuration.
+     * The behavior in case of misalignment depends on the provided {@code BootstrapConfig.misalignedPlatformImports()}:
```

Comment:
```suggestion
     * The behavior in case of misalignment depends on the provided {@link BootstrapConfig#misalignedPlatformImports()}:
```
In case you want to link to a specific API.